kinetic-client.opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+name: "kinetic-client"
+version: "0.0.10"
+maintainer: "Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+authors: "Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+homepage: "https://github.com/openvstorage/kinetic-ocaml-client"
+bug-reports: "https://github.com/openvstorage/kinetic-ocaml-client"
+dev-repo: "https://github.com/openvstorage/kinetic-ocaml-client.git"
+license: "LGPL"
+build: [
+  [make]
+]
+install: [make "install-lib"]
+remove: ["ocamlfind" "remove" "kinetic-client"]
+depends: [
+  "ocamlfind" {build}
+  "ocaml-protoc" {build & >= "1.2.0"}
```

Comment:
I think this is more than a build dependency, as it also contains a runtime lib. I may be wrong though.